src/envs/__init__.py

```diff
@@ -67,6 +69,8 @@ def create_new_env(name: str, do_cache: bool = False) -> BaseEnv:
         env = BehaviorEnv()  # pragma: no cover
     elif name == "repeated_nextto":
         env = RepeatedNextToEnv()
+    elif name == "pybullet_blocks":
+        env = PyBulletBlocksEnv()
```

Comment:
Just a comment for the future if we have lots and lots of environments.

Might be worth walking through all the modules in the `src/envs` directory and automatically discovering subclasses of `BaseEnv`, rather than having to manually add it here every single time a new environment is created?

Reply:
yeah, i'm in favor of this! feel free to open a PR if you know how to do it :)

Reply:
one problem will be the string names here; how would we handle that in your proposal?

Reply:
We could add an abstract class method to BaseEnv and use that to compare strings

Reply:
that would work, but may be just as annoying as the current solution? we can let @tomsilver tiebreak

Reply:
Something like this (simplified) should work and only require giving each env a name. `discover_environments` would discover any new environments so I would think it's slightly easier.

```python
for env in discover_environments():
    if name == env.name():
        return env
```
    
That being said all this isn't a big issue

Reply:
I like @williamshen-nz 's proposal, it's nice to let environments have names anyway.

Reply:
btw, we may want to use a similar pattern elsewhere, e.g., `create_approach`